plugin/kubernetes/xfr.go

```diff
@@ -1,17 +1,80 @@
 package kubernetes
 
 import (
-	"time"
+	"strings"
 
+	"github.com/coredns/coredns/plugin/etcd/msg"
 	"github.com/coredns/coredns/request"
+
+	"github.com/miekg/dns"
+	api "k8s.io/client-go/pkg/api/v1"
 )
 
 // Serial implements the Transferer interface.
-func (e *Kubernetes) Serial(state request.Request) uint32 {
-	return uint32(time.Now().Unix())
-}
+func (k *Kubernetes) Serial(state request.Request) uint32 { return uint32(k.APIConn.Modified()) }
```

Comment:
ah you do it here. Ok, maybe `Modified()` does make sense; however the name 'modified` is somewhat misleading